src/include/verify.h

```diff
@@ -0,0 +1,27 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WT_VERIFY_INFO -- A structure to hold all the information related to a verify operation.
+ */
+struct __wt_verify_info {
+    WT_SESSION_IMPL *session;
+
+    const char *tag;           /* Identifier included in error messages */
+    const WT_PAGE_HEADER *dsk; /* The disk header for the page being verified */
+    WT_ADDR *page_addr;        /* An item representing a page entry being verified */
+    size_t page_size;
+    uint32_t cell_num; /* The current cell offset being verified */
+    uint64_t recno;    /* The current record number in a column store page */
+
```

Comment:
Is it necessary to add an entirely new header file? The `WT_PAGE_HEADER` and `WT_ADDR` both reside in `btmem.h` so it seems like this structure could be there too.